python/ray/experimental/data/read_api.py

```diff
@@ -14,6 +14,16 @@
 from ray.experimental.data.impl.arrow_block import ArrowBlock, ArrowRow
 
 
+def autoinit_ray(f):
+    def wrapped(*a, **kw):
+        if not ray.is_initialized():
+            ray.init()
```

Comment:
```suggestion
            ray.client.connect()
```

Reply:
I don't think we should use that by default yet, it is likely to be removed and is slower.

Reply:
Wdym? It should go down the ray.init code path if no args or auto is specified.